modules/gapi/misc/python/test/gapi_sample_pipelines.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
please name file as `test_*.py`. Otherwise it's not executed: https://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/27871/steps/test_python3/logs/stdio

Reply:
Fixed